lib/datasource/aws-machine-image/index.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Why?

Reply:
sorry - artifact of my early development. Of course this can be removed! Thank you!